src/Apps/Order/Routes/__tests__/Offer.test.tsx

```diff
@@ -81,6 +79,35 @@ describe("Offer InitialMutation", () => {
       console.error = errorLogger
     })
 
+    it("doesn't let the user continue if they haven't typed anything in", () => {
+      const component = getWrapper(testProps)
+
+      expect(component.find(OfferInput).props().showError).toBe(false)
+
+      component.find(Button).simulate("click")
+
+      expect(component.find(OfferInput).props().showError).toBe(true)
+
+      expect(commitMutation).not.toHaveBeenCalled()
+    })
+
+    it("doesn't let the user continue if the offer value is not positive", () => {
+      const component = getWrapper(testProps)
+
+      component
+        .find(OfferInput)
+        .props()
+        .onChange(0)
+
+      expect(component.find(OfferInput).props().showError).toBe(false)
+
+      component.find(Button).simulate("click")
+
+      expect(component.find(OfferInput).props().showError).toBe(true)
```

Comment:
Instead of inspecting the props here, can we assert that an error message shows up with the text we expect?

Reply:
done :+1: